common/db/buffered_bulk.go

```diff
@@ -15,6 +15,9 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+// The default value of maxMessageSizeBytes
+const MAX_MESSAGE_SIZE_BYTES = 48000000
```

Comment:
```suggestion
// The default value of maxMessageSizeBytes
// See: https://docs.mongodb.com/manual/reference/command/hello/#mongodb-data-hello.maxMessageSizeBytes
const MAX_MESSAGE_SIZE_BYTES = 48000000
```
Could we just add this link below the comment? I always forget where this number comes from 😅